src/components/Header/ChainConnectivityWarning.tsx

```diff
@@ -0,0 +1,72 @@
+import { Trans } from '@lingui/macro'
+import { CHAIN_INFO, L2ChainInfo } from 'constants/chains'
+import { useActiveWeb3React } from 'hooks/web3'
+import { AlertOctagon } from 'react-feather'
+import styled from 'styled-components/macro'
+import { ExternalLink, MEDIA_WIDTHS } from 'theme'
+
+const BodyRow = styled.div`
+  color: black;
+  font-size: 12px;
+`
```

Comment:
nit. styling -- idk that we actually follow this but I've always seen a newline separating styled components in our repo

Reply:
ditto below

Reply:
Yeah we dont have a formal guideline on that but thats the standard ive been following